utils/defs.hpp.in

```diff
@@ -43,28 +43,11 @@
 #define UTILS_PURE @ATTRIBUTE_PURE@
 
 
-/// Attribute to mark an entity as unused.
-#define UTILS_UNUSED @ATTRIBUTE_UNUSED@
```

Comment:
I'm surprised to see this go, given that this is different than the parameter issue and it's still used. Why?

Reply:
Ah, shoot.. I didn't realize that :/.